web/src/activity.js

```diff
@@ -121,6 +122,8 @@ export function build_user_sidebar() {
 
     buddy_list.populate({keys: user_ids});
 
+    list_widget.toggle_empty_list_message(buddy_list.$container, filter_text);
+
```

Comment:
The idea of reaching into `list_widget` to get this functionality feels like it breaks the encapsulation of `buddy_list`, which already has its own separate widget for building lists.  I'm not saying it's **definitely** a bad strategy, but it's worthy of debate.  If there's a somewhat simple way to just extend `buddy_list` code to support this paradigm, even with a tiny bit of code duplication, I think I would prefer things to be decoupled here.

Reply:
BTW I don't think any discussion here should block the PR, as it can always be addressed after the fact.